docusaurus/docs/reactnative/ui-components/input_buttons.mdx

```diff
@@ -20,97 +20,65 @@ import HasImagePicker from '../common-content/core-components/channel/props/has_
 import MoreOptionsButton from '../common-content/core-components/channel/props/more_options_button.mdx'
 import UploadsEnabled from '../common-content/core-components/channel/props/uploads_enabled.mdx'
 
-`InputButtons` is a UI component to render container for `AttachButton` and `CommandsButton` on left side of input box, within `MessageInput` component.
+A component rendered on the left side of the input box within [`MessageInput`](./message_input.mdx) that contains the `AttachButton` and the `CommandsButton`.
```

Comment:
opinion: The component rendered...

Reply:
I dont really get why this one is so different from the others. Usually we have "Component to render something" but here we just have the position where a the component will be rendered. Any idea?